polygon-digital-carbon/src/utils/CarbonPool.ts

```diff
@@ -3,7 +3,7 @@ import { CarbonPool, CarbonPoolDailySnapshot, PoolDeposit, PoolRedeem } from '..
 import { ZERO_BI } from '../../../lib/utils/Decimals'
 import { dayFromTimestamp } from '../../../lib/utils/Dates'
 import { loadOrCreateCarbonPoolCreditBalanceDailySnapshot } from './CarbonPoolCreditBalance'
-import { loadOrCreateToken } from '../../../lib/utils/Token'
```

Comment:
Is this old function still being used somewhere else? If not it should prob be deleted...

Reply:
@cujowolf 's suggestion I moved the function inside digital-carbon as there were issues with it being in lib. Used below in `loadOrCreateCarbonPool`

Reply:
Right I see that but is the old version in `lib` still there/still being used? Don't see it being removed in this PR

Reply:
Ah ok, removed